api_tests/applications/views/test_application_list.py

```diff
@@ -1,45 +1,46 @@
+import pytest
 import copy
 import mock
 
-from nose.tools import *  # flake8: noqa
-
 from website.models import ApiOAuth2Application, User
 from website.util import api_v2_url
 from website.util import sanitize
-
 from tests.base import ApiTestCase
 from osf_tests.factories import ApiOAuth2ApplicationFactory, AuthUserFactory
 
 
 def _get_application_detail_route(app):
-    path = "applications/{}/".format(app.client_id)
+    path = 'applications/{}/'.format(app.client_id)
     return api_v2_url(path, base_route='/')
 
 
 def _get_application_list_url():
-    path = "applications/"
+    path = 'applications/'
     return api_v2_url(path, base_route='/')
 
+@pytest.fixture()
+def user():
+    return AuthUserFactory()
 
-class TestApplicationList(ApiTestCase):
-    def setUp(self):
-        super(TestApplicationList, self).setUp()
-
-        self.user1 = AuthUserFactory()
-        self.user2 = AuthUserFactory()
+@pytest.mark.django_db
+class TestApplicationList:
 
-        self.user1_apps = [ApiOAuth2ApplicationFactory(owner=self.user1) for i in xrange(3)]
-        self.user2_apps = [ApiOAuth2ApplicationFactory(owner=self.user2) for i in xrange(2)]
+    @pytest.fixture()
+    def user_app(self, user):
+        return ApiOAuth2ApplicationFactory(owner=user)
 
-        self.user1_list_url = _get_application_list_url()
-        self.user2_list_url = _get_application_list_url()
+    @pytest.fixture()
+    def user_list_url(self):
```

Comment:
Minor: could just be `url` since the test class name makes it clear we're testing the ApplicationList endpoint, but this is fine.